apps/table.html

```diff
@@ -72,6 +72,9 @@
 				</li>
 			</ul>
 			<ul class="navbar-nav">
+				<li class="nav-item link" style="font-family: sans-serif;">
+					<a class="nav-link" href="./signup/signup.html">Signup</a>
+				</li>
```

Comment:
I'm a little divided on if table needs a signup page, but I think it makes sense for consistency.

Reply:
We should have a discussion about the registration process before working on the `signup` page.